src/pages/graphics.js

```diff
@@ -0,0 +1,69 @@
+import Page from "./page"
+import Grapher from "../helpers/grapher"
+import Navbar from '../helpers/navbar';
+import Setting, { MetricSettingsBox } from './settings'
+import "../helpers/graph.css"
+import { BsFillCircleFill, BsSuitDiamondFill } from "react-icons/bs";
+//This webpage displays metrics graphs relative to the hashes
+export default class GraphicsPage extends Page {
+  render() {
+    document.body.style.setProperty("--textFontSize", (parseInt(this.state.config.textFontSize)).toString() + "px")
```

Comment:
this seems quite difficult to understand, does this have to be done to the document object?

Reply:
these have been moved to the Page class